pep-0594.rst

```diff
@@ -731,16 +482,6 @@ formatter
 The `formatter <https://docs.python.org/3/library/formatter.html>`_ module
 is an old text formatting module which has been deprecated since Python 3.4.
```

Comment:
I suggest moving this to the `Removed` section, since this was removed on schedule in Python 3.10.

Reply:
Resolved in #2274